modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -27,11 +27,11 @@ In the table, features are marked with the following statuses:
 
 s|Version s|kam  s|Argo CD CLI  s|Helm  s|Kustomize s|Argo CD s|Argo Rollouts s|Dex     s|RH SSO |
 
+|1.17.0 |NA  |3.0.11 TP |3.17.1 GA |5.6.0 GA |3.0.11 GA |1.8.3 GA |2.41.1 GA |7.6.0 GA |4.12-4.19
+
 |1.16.0 |NA  |2.14.7 TP |3.16.4 GA |5.4.3 GA |2.14.4 GA |1.8.0 GA |2.41.1 GA |7.6.0 GA |4.12-4.18
 
 |1.15.0 |NA  |2.13.1 TP |3.15.4 GA |5.4.3 GA |2.13.1 GA |1.7.2 GA |2.41.1 GA |7.6.0 GA |4.14-4.17
-
-|1.14.0 |0.0.51 TP |2.12.3 TP |3.15.2 GA |5.4.2 GA |2.12.3 GA |1.7.1 GA |2.39.1 GA |7.6.0 GA |4.12-4.17
 |===
```

Comment:
Can we add a column for `argocd-agent` and put the version as `0.2.0 TP` (Technical Preview) just for 1.17.0. For other versions, leave the value empty or NA.

Reply:
Fixed.